src/components/LicenseDetailsCard.vue

```diff
@@ -50,7 +50,7 @@
             target="_blank"
             rel="noopener noreferrer"
         >
-            See the License Deed
+            See the CC0 Deed
```

Comment:
Actually, there was a bug here: all the texts should be added through the $t('text-key') to enable internationalization instead of just the string as you can see here. 
So, can you please add two i18n strings: one saying 'See the License Deed', and the other saying 'See the CC0 Deed', and render the correct one? When the selectedLicense is CC0, it should show the CC0 string, and in other cases, it should show the License string.